cmd/gcp-controller-manager/app/gke_approver.go

```diff
@@ -128,6 +131,13 @@ func (a *gkeApprover) handle(csr *capi.CertificateSigningRequest) error {
 			glog.Infof("validator %q: SubjectAccessReview approved for CSR %q", r.name, csr.Name)
 			csrApprovalStatus.WithLabelValues("approve", r.authFlowLabel).Inc()
 			csrApprovalLatency.WithLabelValues("approve", r.authFlowLabel).Observe(time.Since(start).Seconds())
+			if r.precondition != nil {
```

Comment:
Move this before `csrApprovalStatus.WithLabelValues("approve", r.authFlowLabel).Inc()` above.
Also, log on error and success.